Nos/Views/ProfileHeader.swift

```diff
@@ -131,6 +142,40 @@ struct ProfileHeader: View {
                     followsResult: followsResult,
                     followersResult: followersResult
                 )
+                HStack {
```

Comment:
I don't know the codebase well enough yet, but would it make sense to move this out so body is smaller? In the past, I've done something like this:

```
var profileHeaderTab: some View {
    HStack { 
        ...
```

And then the body becomes smaller since right here you can just use `profileHeaderTab`.

Of course, you could just create a new type for ProfileHeaderTab if that makes sense. This is a step short of that and therefore easier and lower impact, but still gives the advantage of making the body smaller and more readable. Thoughts?